packages/cardpay-cli/README.md

```diff
@@ -59,3 +62,42 @@ ARGUMENTS
   NETWORK   The network to use ("sokol" or "xdai")
   MNEMONIC  (Optional) Phrase for mnemonic wallet. Also can be pulled from env using MNEMONIC_PHRASE
 ```
+
+## `yarn cardpay usd-price <TOKEN> <AMOUNT> --network=NETWORK [--mnemonic=mnmonic]`
+Get the USD value for the specified token name in the specified amount. This returns a floating point number in units of USD.
+```
+USAGE
+  $ yarn cardpay usd-price <TOKEN> <AMOUNT> --network=NETWORK [--mnemonic=MNEMONIC]
+
+ARGUMENTS
+  TOKEN     The token symbol (without the .CPXD suffix)
+  AMOUNT    The amount of the specified token (not in units of wei)
+  NETWORK   The network to use ("sokol" or "xdai")
+  MNEMONIC  (Optional) Phrase for mnemonic wallet. Also can be pulled from env using MNEMONIC_PHRASE
+```
+
+## `yarn cardpay eth-price <TOKEN> <AMOUNT> --network=NETWORK [--mnemonic=mnmonic]`
+Get the ETH value for the specified token name in the specified amount (in units `ether`).
+```
+USAGE
+  $ yarn cardpay eth-price <TOKEN> <AMOUNT> --network=NETWORK [--mnemonic=MNEMONIC]
+
+ARGUMENTS
+  TOKEN     The token symbol (without the .CPXD suffix)
+  AMOUNT    The amount of the specified token (not in units of wei)
+  NETWORK   The network to use ("sokol" or "xdai")
+  MNEMONIC  (Optional) Phrase for mnemonic wallet. Also can be pulled from env using MNEMONIC_PHRASE
+```
+
+## `yarn cardpay oracle-updated-at <TOKEN> --network=NETWORK [--mnemonic=mnmonic]`
+This returns the date that the oracle was last updated for the specified token.
```

Comment:
Is this the only oracle we are likely to be concerned with? If not, perhaps `exchange-oracle-updated-at` or `price-oracle-updated-at` would be clearer